web/satellite/src/main.ts

```diff
@@ -2,11 +2,33 @@ import Vue from 'vue';
 import App from './App.vue';
 import router from './router';
 import store from './store';
+import {HttpLink} from "apollo-link-http";
+import ApolloClient from "apollo-client/ApolloClient";
+import {InMemoryCache} from "apollo-cache-inmemory";
+import VueApollo from "vue-apollo";
 
 Vue.config.productionTip = false;
 
+const httpLink = new HttpLink({
```

Comment:
not sure that httpLink is a best name.
Maybe satelliteUrl, satelliteEndpoint, gqUrl?
And place it in a better place, i'm pretty sure that it will be used in multiple view models